manual/Makefile

```diff
@@ -1,9 +1,17 @@
 # Makefile for the reference and user manuals
 TEXDIR=tex_files
 VPATH=$(TEXDIR)
-.PHONY:all
+.PHONY:all clean sphinx
 
-all: user_manual.pdf \
+all: sphinx
+
+sphinx:
+	sphinx-build -b latex sphinx/ build/
+	cd build && latexmk -pdf BOUT
+	test -e BOUT.pdf || ln -s build/BOUT.pdf .
+	@echo "Documentation is available in BOUT.pdf"
+
+old: user_manual.pdf \
```

Comment:
This looks good, but it means the instructions for building the documents that were introduced as a part of this PR are no longer correct ([see here](https://github.com/boutproject/BOUT-dev/pull/674/commits/aa0100104659729eb9b6adb7ccfcc0eeeb342cf0#diff-e2c8a2035737632f4939660d46e748f7R21)). Similarly it might worth adding a note to the manual that there's a simple make target for building the sphinx documentation as well as manually typing the commands given.